Tribler/Test/test_as_server.py

```diff
@@ -301,6 +302,7 @@ class TestGuiAsServer(TestAsServer):
 
     def setUp(self):
         AbstractServer.setUp(self, annotate=False)
+        self.setUpPreSession()
```

Comment:
This shouldn't be neccesary, we're using the session from Tribler -> and its default settings.